src/ImageRunModal.jsx

```diff
@@ -935,15 +935,17 @@ export class ImageRunModal extends React.Component {
                                 <Checkbox id="run-image-dialog-memory-limit-checkbox"
                                   isChecked={this.state.memoryConfigure}
                                   onChange={checked => this.onValueChanged('memoryConfigure', checked)} />
-                                <TextInput type='number'
+                                <NumberInput
                                    value={dialogValues.memory}
                                    id="run-image-dialog-memory"
-                                   className="dialog-run-form-input"
-                                   step={1}
                                    min={0}
-                                   isReadOnly={!this.state.memoryConfigure}
+                                   isDisabled={!this.state.memoryConfigure}
                                    onClick={() => !this.state.memoryConfigure && this.onValueChanged('memoryConfigure', true)}
-                                   onChange={value => this.onValueChanged('memory', value)} />
+                                   onPlus={() => this.onPlusOne('memory')}
+                                   onMinus={() => this.onMinusOne('memory')}
+                                   minusBtnAriaLabel={_("Decrease memory by one")}
+                                   plusBtnAriaLabel={_("Increase memory by one")}
```

Comment:
"by one" sounds strange -- if we can't easily add the unit here, maybe just drop the "by one"?

Reply:
removed